src/base_alloc/base_alloc_linear.c

```diff
@@ -88,7 +88,11 @@ umf_ba_linear_pool_t *umf_ba_linear_create(size_t pool_size) {
         pool_size = MINIMUM_LINEAR_POOL_SIZE;
     }
 
-    pool_size = ALIGN_UP(pool_size, ba_os_get_page_size());
+    pool_size = ALIGN_UP_SAFE(pool_size, ba_os_get_page_size());
+    if (pool_size == 0) {
+        LOG_ERR("umf_ba_linear_create: pool_size page alignment overflow");
```

Comment:
`LOG_ERR("umf_ba_linear_create: ` - I am almost sure, that LOG_* functions always prints out the function prefix, so it is not needed.

Reply:
Removed prefixes